documentation/modules/ROOT/pages/connectors/sqlserver.adoc

```diff
@@ -2160,8 +2160,6 @@ For `read_uncommitted` there are no data consistency guarantees at all (some dat
 a|String that represents the criteria of the attached timestamp within the source record (ts_ms).
 
 * `commit` (default) sets the source timestamp to the time when the record was committed to the database.
-* `processing` sets the source timestamp to the time when {prodname} accesses the record in the change table.
-Use the `processing` option if you want {prodname] to set the top level `ts_ms` value, or if you want to avoid the additional cost of {prodname} querying the database to extract the LSN timestamps.
```

Comment:
I was thinking of removing the entire option from the docs, there should be no need to specify it at all, right?

Reply:
I see. Makes sense.

Reply:
Yes, to be sure: the code for this should still remain for the time being. We only don't want to point users to this option any longer.

Reply:
Right, the code will remain in place for now.